tests/Controllers/SessionsControllerTest.php

```diff
@@ -27,12 +27,15 @@ public function testLogin()
     public function testLoginInactiveUser()
     {
         $password = 'password1';
-        $user = factory(User::class)->create(compact('password'));
+        $country_acronym = factory(\App\Models\Country::class)->create(['acronym' => '__'])->acronym;
```

Comment:
- no need to specify the acronym
- variable names should be camelCased
- import the class instead of specifying full name

Reply:
> no need to specify the acronym

If I don't specify the acronym it gives duplicate error for me because countries exist in my case, that's the whole point of this PR.

> variable names should be camelCased

Compact won't work with camelCase, do you want me to ditch the compact here?

Reply:
> If I don't specify the acronym it gives duplicate error for me because countries exist in my case, that's the whole point of this PR.

```php
(Country::first() ?? factory(Country::class)->create())->getKey()
```

> Compact won't work with camelCase, do you want me to ditch the compact here?

yes